zebrad/src/components/tracing/component.rs

```diff
@@ -266,7 +270,23 @@ impl Tracing {
             info!("activated progress bar");
 
             if config.log_file.is_some() {
-                eprintln!("waiting for initial progress reports...");
+                // Clear screen
+                eprint!("\x1B[2J");
+                {
+                    eprintln!(
+                        "{}",
+                        std::str::from_utf8(&ZEBRA_ART)
+                            .expect("should always work on a UTF-8 encoded constant")
+                    );
+                }
+                eprintln!(
```

Comment:
These logs can be sent regardless of whether the output is a terminal.

Reply:
Good point. Did that on da446250;